core/domain/feedback_jobs_one_off.py

```diff
@@ -72,32 +72,35 @@ def reduce(key, stringified_message_ids):
 class FeedbackSubjectOneOffJob(jobs.BaseMapReduceOneOffJobManager):
     """One-off job for updating the feedback subject."""
 
-    DEFAULT_SUBJECT = '(Feedback from a learner)'
+    DEFAULT_SUBJECT = u'(Feedback from a learner)'
 
     @classmethod
     def entity_classes_to_map_over(cls):
         return [feedback_models.FeedbackThreadModel]
 
     @staticmethod
     def map(item):
-        if item.subject != FeedbackSubjectOneOffJob.DEFAULT_SUBJECT:
+        yield (item.id, item.subject)
+
+    @staticmethod
+    def reduce(key, stringified_subject):
```

Comment:
Why are you doing all this stuff in reduce? Wouldn't it be better to do it in map instead? That way it doesn't have to go through the shuffle stage.